seth.go

```diff
@@ -415,7 +415,8 @@ type Transaction struct {
 	Input       Data     `json:"input"`            // input data
 }
 
-type request struct {
+// RPCRequest is a request to be sent to an RPC server.
+type RPCRequest struct {
```

Comment:
These types don't need to be exported for `json.Marshal` to work. Only the fields need to be exported.